jscomp/syntax/tests/printer/expr/expected/jsx.res.txt

```diff
@@ -4,8 +4,8 @@ let x = <Foo.Bar className="container" />
 let x = <Foo.Bar.Baz className="container" />
 let x = <Foo.bar className="container" />
 let x = <Foo.baz className="multiline" />
-let x = <\"custom-tag" className="container" />
-let x = <Foo.\"custom-tag" className="container" />
+let x = <custom-tag className="container" />
+let x = <Foo.custom-tag className="container" />
```

Comment:
`<custom-tag` is now allowed and should not be printed as `<\"custom-tag"` exotic identifier, hence the update